app/frontend/wallet/helpers/tokenFormater.ts

```diff
@@ -35,9 +35,11 @@ export const aggregateTokenBundles = (tokenBundle: TokenBundle[]): TokenBundle =
 
 export const groupTokenBundleByPolicyId = (
   tokenBundle: TokenBundle
-): {[policyId: string]: TokenBundle} => {
+): {[policyId: string]: Omit<Token, 'policyId'>[]} => {
   return _(tokenBundle)
+    .orderBy(['policyId', 'assetName'], ['asc', 'asc'])
     .groupBy(({policyId}) => policyId)
+    .mapValues((tokens) => tokens.map(({assetName, quantity}) => ({assetName, quantity})))
```

Comment:
This is rather a hotfix solution (I experienced some bugs with different token orders) and I think we should make it more explicit how we sort the assets and address this in a separate PR.

Reply:
I think that an explanatory comment why order matters and bunch of unit tests covering that behavior would actually do in this case

Reply:
ah wait, but this is returning an object - this doesn't really give any reasonable guarantees when it comes to keys ordering. It would make sense then to create a separate helper function which transforms the token bundle into a list of tuples - the format that both Ledger and Trezor accept

Reply:
Thnx, fixed in another PR.